src/gui/cli/Player.cpp

```diff
@@ -64,10 +62,29 @@ namespace Zappy::GUI {
 
     int Player::getInventory(long unsigned int aSlot) const
     {
-        if (aSlot > 6) {
-            return -1;
+        switch (aSlot) {
```

Comment:
you can avoid switch by creating an array containing all items. then you can return `array[aSlot]`